### PR TITLE
Add SR2019 Kickstart events

### PR DESCRIPTION
**Pull request by PeterJCLaw**
These are based on the SR2017 events, updated for the information which has been confirmed so far.

### CODE REVIEW

**Comment by RealOrangeOne** [comment]
Question: Should we ship these in their current state, which whilst missing information, are correct. This would mean we could link teams to them for more information, and we can just update them as time goes on.

**Comment by PeterJCLaw** [comment]
I'd rather populate them, ideally this week, and emit a blog post announcing them.

File: _events/sr2019/southampton-kickstart.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+date: 2018-11-10 10:00:00
+layout: event
+type: kickstart
+location: Southampton University
+---
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced and robotics kits will be handed out to the teams.
+
+**Any team is welcome to attend the Southampton Kickstart, however you need to
```
Comment (RealOrangeOne) [inline _events/sr2019/southampton-kickstart.md]: I'm not sure if this is entirely necessary? Seeing as the event is somewhat invite-only, and that the London event is _very_ invite-only, we can assume where people are going. We have MailChimp to tell us who's going where.

Comment (PeterJCLaw) [reply _events/sr2019/southampton-kickstart.md]: ca1618b0556399947fdd64b6ec32c44e296f35a8

File: _events/sr2019/southampton-kickstart.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+| Time  | Info |
+|-------|------|
+| 10:00 | Introductory presentation, where the game of SR2019 will be announced. |
+| 11:00 | Team kits will be handed out, and teams will take part in activities to get them acquainted with their kits. |
+| 12:30 | There will be a lunch break. Teams are expected to bring their own lunches or get them from food vendors near the venues. |
+| 17:00 | Finish. |
+
+[teams-contact]: mailto:teams@studentrobotics.org
+[soton-food-map]: TODO
```
Comment (RealOrangeOne) [inline _events/sr2019/southampton-kickstart.md]: For this and the below links, best copy the links from https://studentrobotics.org/events/sr2017/southampton-tech-day-february/.

Comment (PeterJCLaw) [reply _events/sr2019/southampton-kickstart.md]: In theory yes, however they say 2017 all over them.

Comment (Adimote) [reply _events/sr2019/southampton-kickstart.md]: I've made a new map for you here: https://drive.google.com/open?id=1vDqr-yQh-OO7MzVa_OiuKD5xUrEsLprr&usp=sharing

File: _events/sr2019/london-kickstart.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+Whitechapel,  
+London,  
+E1 8DE  
+
+The easiest way to travel will be by public transport -- Aldgate, Aldgate East
+and Tower Gateway stations are all within a five minute walk, and Liverpool
+Street station is a ten minute walk away. There are also a large number of
+busses which stop nearby.
+
+<!-- TODO: find out whether we can provide parking
```
Comment (RealOrangeOne) [inline _events/sr2019/london-kickstart.md]: Has this TODO been done?

Comment (PeterJCLaw) [reply _events/sr2019/london-kickstart.md]: Sort of (see https://github.com/srobo/website/pull/91#issuecomment-429630409 for details).
I believe we would be able to provide parking, I don't know what the vehicle size limit would be. I'll check that on Monday, but I suspect anyone who really wants to drive would ask us about it.

**Pull request merged**